### PR TITLE
ref(open-pr-comments): remove from EA list and add javascript/typescript

### PR DESCRIPTION
**Pull request by cathteng**
- Removes EA banner and Open PR Comments from the list of EA features
- Adds note about Javascript/Typescript

### CODE REVIEW

**Review by Dhrumil-Sentry — APPROVED** [review]

File: docs/product/integrations/source-code-mgmt/github/index.mdx [hunk 1]
```diff
@@ -261,15 +261,13 @@ Sentry will only comment on pull requests less than two weeks old that are assoc
...
-<Include name="feature-available-for-user-group-early-adopter.mdx" />
-
 When a pull request is opened, Sentry will parse the filenames and function names modified in the pull request from the PR diff to find recent unhandled, unresolved issues associated with your pull request.
 
 ![Sentry comment on open pull request in GitHub](/product/integrations/source-code-mgmt/github/sentry-github-open-pr-comment.png)
 
 Sentry will comment on the pull request with up to five issues per file that were first seen within the past 90 days and were last seen within the past 14 days.
 
-This feature requires [code mappings](/product/issues/suspect-commits/#2-set-up-code-mappings) and is currently only supported for Python files.
+This feature requires [code mappings](/product/issues/suspect-commits/#2-set-up-code-mappings) and is currently only supported for Python and Javascript/Typescript files. For Javascript/Typescript, please ensure that you've unminified your code in Sentry.
```
Comment (sentaur-athena) [inline docs/product/integrations/source-code-mgmt/github/index.mdx:270]: noob question 😅 do everyone know how to unminify code in Sentry? If not I think we need a link to a support doc here for that.

**Review by vivianyentran — APPROVED** [review]
looks good!

File: docs/product/integrations/source-code-mgmt/github/index.mdx (hunk 1 above)
Comment (cathteng) [reply docs/product/integrations/source-code-mgmt/github/index.mdx:270]: there's usually a button in the stacktrace... let me try to find an example

Comment (sentaur-athena) [reply docs/product/integrations/source-code-mgmt/github/index.mdx:270]: Don't worry too much. It might be just something that I'm not familiar with.

Comment (cathteng) [reply docs/product/integrations/source-code-mgmt/github/index.mdx:270]: actually the correct phrasing should be "make sure to configure source maps for unminified code" or something
Screenshot 2024-02-09 at 11 59 44

**Pull request merged**